.github/workflows/test.yml

```diff
@@ -353,3 +355,84 @@ jobs:
           kernel-root: '.'
           max-cpu: 8
           kernel-test: ${{ matrix.test }}
+  veristat:
+    name: veristat
+    needs: [set-matrix, build]
+    runs-on: ${{ fromJSON(needs.set-matrix.outputs.veristat-runs-on) }}
+    timeout-minutes: 100
+    env:
+      KERNEL: LATEST
+      REPO_ROOT: ${{ github.workspace }}
+      REPO_PATH: ""
+      KBUILD_OUTPUT: kbuild-output/
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: vmlinux-x86_64-gcc
+          path: .
+      - name: Untar artifacts
+        # zstd is installed by default in the runner images.
+        run: zstd -d -T0  vmlinux-x86_64-gcc.tar.zst --stdout | tar -xf -
+
+      - name: Prepare rootfs
+        uses: libbpf/ci/prepare-rootfs@main
+        with:
+          project-name: 'libbpf'
+          arch: x86_64
+          kernel: LATEST
+          kernel-root: '.'
+          kbuild-output: ${{ env.KBUILD_OUTPUT }}
+          image-output: '/tmp/root.img'
+          test: run_veristat
+
+      - name: Run veristat
+        uses: libbpf/ci/run-qemu@main
+        timeout-minutes: 10
+        with:
+          arch: x86_64
+          img: '/tmp/root.img'
+          vmlinuz: '${{ github.workspace }}/vmlinuz'
+          kernel-root: '.'
+          max-cpu: 8
+          kernel-test: run_veristat
+          output-dir: '${{ github.workspace }}'
+
+      # veristat.csv is produced by run-qemu run_veristat action
+      - uses: actions/upload-artifact@v3
+        with:
+          name: x86_64-gcc-veristat-log
+          if-no-files-found: error
+          path: '${{ github.workspace }}/veristat.csv'
+
+      # For pull request:
+      # - get baseline log from cache
+      # - compare it to current run
+      - if: ${{ github.event_name == 'pull_request' }}
+        uses: actions/cache/restore@v3
+        with:
+          key: x86_64-gcc-veristat-baseline
+          restore-keys: |
+            x86_64-gcc-veristat-baseline-
+          path: '${{ github.workspace }}/veristat-baseline.csv'
+
+      - if: ${{ github.event_name == 'pull_request' }}
+        name: Show veristat comparison
+        run: |
+          set -e
```

Comment:
Unnecessary? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Reply:
Unnecessary indeed, removed.